tools/block-generator/runner/run.go

```diff
@@ -112,16 +126,44 @@ func (r *Args) run() error {
 		}
 	}()
 
-	//indexerShutdownFunc, err := startIndexer(dataDir, logfile, r.LogLevel, r.IndexerBinary, algodNet, indexerNet, r.PostgresConnectionString, r.CPUProfilePath)
-	//if err != nil {
-	//	return fmt.Errorf("failed to start indexer: %w", err)
-	//}
-	//defer func() {
-	//	// Shutdown indexer
-	//	if err := indexerShutdownFunc(); err != nil {
-	//		fmt.Printf("Failed to shutdown indexer: %s\n", err)
-	//	}
-	//}()
+	// get conduit config template
+	cwd, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("current working directory: %w", err)
+	}
+	t, err := template.ParseFiles(path.Join(cwd, "data/conduit.yml"))
+	if err != nil {
+		return fmt.Errorf("unable to open config template file: %w", err)
+	}
```

Comment:
opinionated nit - feel free to ignore: I think it would be slightly cleaner to embed the config template here instead having to read it at run time.

If you renamed `conduit.yml` to `conduit.yml.tmpl` you could have a directive:
```go
//go:embed ../data/conduit.yml.tmpl
var conduitConfigTmpl string
```
and instead of calling `os.Getwd()` and `template.ParseFiles` would do something like:

```go
	t, err := template.New("code").Parse(conduitConfigTmpl)
	if err != nil {
		return fmt.Errorf("unable to parse conduit config template: %w", err)
	}
```